File: AndoirdApp/app/src/main/java/esi/tdm/mypharmacy/entity/Pharmacy.kt
package esi.tdm.mypharmacy.entity

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey
import java.io.Serializable


@Entity(tableName = "pharmacies")
data class Pharmacy(
    @PrimaryKey
    var id: Int,
    var city_id: Int,
    var longitude: Int,
    var latitude: Int,
    var name: String? = "",
    var address: String? = "",
    var phone: String? = "",
    var check_in: String? = "",
    var check_out: String? = "",
    var facebook_page: String? = ""
) : Serializable

File: BackendNode/models/pharmacy.js
'use strict';
module.exports = (sequelize, DataTypes) => {
  const Pharmacy = sequelize.define('Pharmacy', {
    name: DataTypes.STRING,
    city_id: DataTypes.STRING,
    longitude: DataTypes.STRING,
    latitude: DataTypes.STRING,
    check_in: DataTypes.STRING,
    check_out: DataTypes.STRING,
    facebook_page: DataTypes.STRING,
    phone: DataTypes.STRING,
    address: DataTypes.STRING
  }, {});
  Pharmacy.associate = function(models) {
    // associations can be defined here
  };
  return Pharmacy;
};
